plyer/platforms/android/gravity.py

```diff
@@ -52,26 +52,30 @@ def onAccuracyChanged(self, sensor, accuracy):
 
 class AndroidGravity(Gravity):
 
-    listener = None
+    def __init__(self):
+        super(AndroidGravity, self).__init__()
+        self.state = False
 
     def _enable(self):
-        if not self.listener:
+        if (not self.state):
```

Comment:
it will work fine without brackets. Its just my habit of putting brackets with if condition. It looks more simplified to me. Does it violate any rule of coding style?